AwsCryptographyPrimitives/src/Digest.dfy

```diff
@@ -13,10 +13,9 @@ module Digest {
   function method Length(digestAlgorithm: Types.DigestAlgorithm): nat
   {
     match digestAlgorithm
-    case SHA_512 => 64
-    case SHA_384 => 48
-    case SHA_256 => 32
-    case SHA_1 => 20
```

Comment:
do we not use sha1 length anywhere?
I am not sure if some read only algorithm suites use sha1

Reply:
It's worse than that. SHA_1 is not defined.
In Dafny, the we're declaring a new variable named "SHA_1" to handle all the rest of the cases.
Since the three above are all that there is, there never was a "the rest" to be handled.
If SHA_1 existed, then that match statement would not verify, as there would be an unhandled option.